serverless.yml

```diff
@@ -22,9 +22,16 @@ package:
   - './bin/**'
 
 functions:
-  cron:
+  maintenance:
     handler: bin/maintenance
     events:
     # cron(Minutes Hours Day-of-month Month Day-of-week Year)
     # Either `day-of-month` or `day-of-week` must be a question mark (?)
     - schedule: cron(0 4 ? * SUN *)
+
+  outdatedRequests:
```

Comment:
I don't understand why this needs to be a separate lambda since it's also running once a week.

Reply:
Having it separate gives us freedom to run it on a different schedule than the other jobs. Although it's set for a weekly run here, maybe in the future we will want to change that.